CHANGELOG.md

```diff
@@ -52,6 +52,7 @@
 - Adjust background color of OuiToolTip in `next` theme ([#1004](https://github.com/opensearch-project/oui/pull/1004))
 - Add new `middle-out` order prop option to `OuiPaletteColorBlind` ([#856](https://github.com/opensearch-project/oui/pull/856))
 - Add new icons for OpenSearch Dashboards v2.10.0 ([#1014](https://github.com/opensearch-project/oui/pull/1014))
+- Match .browserlist with OpenSearch Dashboards ([#1023](https://github.com/opensearch-project/oui/issues/1023)) 
```

Comment:
This should go in the unreleased section

Reply:
Done!